playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -266,8 +266,9 @@
   - include_tasks: docker/tasks/upgrade.yml
     when: l_docker_upgrade is defined and l_docker_upgrade | bool and not openshift_is_atomic | bool
 
+
 - name: Drain and upgrade master nodes
-  hosts: oo_masters_to_config:&oo_nodes_to_upgrade
+  hosts: "{{ l_double_upgrade_cp | default(False) | ternary('all:!all', 'oo_masters_to_config:&oo_nodes_to_upgrade') }}"
```

Comment:
Let's add a comment why were doing this here.

Reply:
Done